src/Phinx/Config/Config.php

```diff
@@ -294,6 +304,52 @@ public function getTemplateClass()
      }
 
     /**
+     * Get the version order.
+     *
+     * @return string
+     */
+    public function getVersionOrder()
+    {
+        if (!isset($this->values['version_order'])) {
+            // if the configuration value is missing, return the default version order (creation time)
+            return self::VERSION_ORDER_CREATION_TIME;
```

Comment:
initially I only had the getter, which is why the logic ended up here, will move it to the setter